docs/UserGuide.md

```diff
@@ -182,26 +184,26 @@ Format: `find k/KEYWORD [k/MORE_KEYWORDS]…​ [f/ATTRIBUTE_FIELD]`
 `ATTRIBUTE_FIELD` can take on the following values
 `all`, `avail`, `appstatus`, `course`, `email`, `intstatus`, `name`, `phone`, `seniority`, `studentid`, `remark`
 
-* A candidate's`seniority` may match any case variation of `COM1`, `COM2`, `COM3` or `COM4`
-* A candidate's `avail` may match any case variation of days in the format of `MON`, `TUE`, `WED`, `THUR` or `FRI`
+* A candidate's`seniority` may match any case variation of `COM1`, `COM2`, `COM3` or `COM4`.
+* A candidate's `avail` may match any case variation of days in the format of `MON`, `TUE`, `WED`, `THU` or `FRI`.
 
 </div>
 
-* The keyword search is case-insensitive. e.g `hans` will match `Hans`
-* The attribute field is case-insensitive. e.g. `NAME` is equivalent to `name`
+* The keyword search is case-insensitive. e.g `hans` will match `Hans`.
+* The attribute field is case-insensitive. e.g. `NAME` is equivalent to `name`.
 * The search will return a list of all candidates containing any of the specified keyword(s) in the specified attribute field.
 * For `f/all`, the search will find keywords across all attribute fields of the candidate records.
 * Only full keywords will be matched
-  e.g. `k/jane doe f/name` will not match candidates with name `jane koe` or just `jane`
+  e.g. `k/jane doe f/name` will not match candidates with name `jane koe` or just `jane`.
 * Candidates matching at least one full keyword (in the specified attribute field) will be returned i.e. OR search,
-  e.g. `k/Jane k/Doe f/name` will return candidates with name e.g. `Jane Koe`, `John Doe`
+  e.g. `k/Jane k/Doe f/name` will return candidates with name `Jane Koe`, `John Doe`.
 * If multiple `ATTRIBUTE_FIELD`s are provided, only the last field will be used.
 * If no `ATTRIBUTE_FIELD` is provided, the search will be conducted across all fields by default.
 
 Examples:
-* `find k/Jane f/name` returns candidates with name e.g. `jane` and `jane doe`
-* `find k/Computer k/Science f/course` returns candidates with the course field i.e. `computer science` and `computer engineering`
-* `find k/Jane k/Tan f/name` returns candidates with name e.g. `Jane`, `tan` and `John Tan`
+* `find k/Jane f/name` returns candidates with name e.g. `jane` and `jane doe`.
+* `find k/Computer k/Science f/course` returns candidates with the course field i.e. `computer science` and `computer engineering`.
```

Comment:
Hi @charmainehly ! Just want to clarify this -> is it supposed to e.g. or i.e. ? (if it matters?) Also, should the example be`Computer Science` and `Computer Engineering` with capitalisation (since our course names are case sensitive)

Reply:
hmm yea i get it though! the use of e.g. and i.e. a bit confusing right :') i was thinking of using i.e. because there are fixed values for the course fields, lemme have a think if i can improve the phrasing in a future PR! :) 

But good point!!!! Hang on, I'll push up a new commit to fix the capitalisation 😄